cafebabel/templates/articles/drafts/create.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% import "articles/macros.html" as macros without context %}
 
+{% block extra_js %}<script src={{ url_for('static', filename='js/edit-article.js') }}></script>{% endblock %}
```

Comment:
Maybe we should better name it `articles-tags.js`, it's kind of weird to have an edit file in the create template 😉